CHANGES.rst

```diff
@@ -38,9 +38,10 @@ nasa_exoplanet_archive
   release. Support for querying old tables (exoplanets, compositepars, and
   exomultpars) has been dropped. [#2067]
 
+atomic
+^^^^^^
 
-Service fixes and enhancements
-------------------------------
```

Comment:
@bsipocz note that this was a redundant header - we might have to remove this in other PRs too

Reply:
oh, do remove it in one PR only, the other may end up in a conflict otherwise.